### PR TITLE
Mint clone

### PR DESCRIPTION
**Pull request by bafiam**
#### What does this PR do?
Create a web page replica of the [mint.com Sign up page](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3Dbd60d597-946f-491e-8dad-e84b0704db4e%26adobe_mc%3DMCMID%253D73771161536357258650054885978666973615%257CMCAID%253D2ED8FCE905158000-400A284D4C7F9B19%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1572018203%26ivid%3Dbd60d597-946f-491e-8dad-e84b0704db4e)
#### How should this be manually tested?
Visit the github pages url <https://fegzycole.github.io/mint-clone/>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+          <div class="form-label"><label for="email">Email address</label></div>
+
+          <input type="email" name="email" id="email">
+          <div class="form-label"><label for="phone">Phone (recommended)</label></div>
+          <input type="text" name="phone" id="phone">
+          <p class="phone-msg">Standard call, messaging or data rates may apply.</p>
+          <div class="form-label"><label for="password">Password</label></div>
+          <div class="password">
+            <input type="password" name="password" id="password"><img class="password-key" src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.2.9/images/lock-left-grey@3x.png">
```
Comment (shshamim63) [inline index.html]: An img element must have an alt attribute.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+.form-submit button {
+  width: 85%;
+  height: 44px;
+  margin-left: 3px;
+  margin-top: 25px;
+  background: #0077c5;
+  border: none;
+  margin-bottom: 35px;
```
Comment (shshamim63) [inline styles.css]: Shorthand properties are CSS properties that let you set the values of multiple other CSS properties simultaneously. Using a shorthand property, you can write more concise (and often more readable) style sheets, saving time and energy.
For margin the sequence is ->
margin : top right bottom left.

**Review by shshamim63 — COMMENTED** [review]
## :point_right: Changes Required! :point_left: 
___
### Hi, @bafiam  And @fegzycole 
Well done, I liked the way you guys handled the flow of the form. Perfectly place the lock icon on the password input field :sunglasses: . 
But there are some fields that you need to cover to get an approval on this project.
___

#### Please follow the instructions below to achieve your approval  :point_down: 
___
## [Mandatory Changes]
1. The original website header consists three different logo with their respective name. But i can see you have covered that part by adding a single image. [How can you verify that?{Ans: Inspect the element of both yours and original page.}]. So please modify this section add three logo/image with their name, place them in one line as the original webpage.
mint

2. I think that the width class .form-div 30% is almost exact to the original page.
3. Shorthand properties are really helpful to save energy and it increases the readability. Please change the commented code. 
___
#### Please modify your code and ask for another code review. :slightly_smiling_face: 
___

## <em>Happy Coding</em> :four_leaf_clover:

### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain )

**Comment by ebukaume** [comment]
## Approved! :champagne: 

Nice work guys. You have nailed! **You can now merge this branch.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**